features/admin_creates_auction.feature

```diff
@@ -2,6 +2,7 @@ Feature: Admin creates auctions in the admins panel
   As an admin
   I should be able to create auctions
 
+  @background_jobs_off
```

Comment:
oy! well maybe we should also make sure the logic in the job doesn't blow up if an auction ends without a winner or losers? can imagine we may someday have an auction with zero bids...